backend/lambdas/api/search_repositories/search_repositories/get.py

```diff
@@ -5,6 +5,9 @@
 from artemisdb.artemisdb.paging import Filter, FilterMap, FilterMapItem, FilterType, PageInfo, apply_filters, page
 
 
+CICD_TOOL_PARAM = "cicd_tool"
```

Comment:
Would it make sense to store this in either `artemisapi.const` or `artemisdb.artemisdb.consts` since the same string is declared here and in `paging.py` (although it's in a list in the latter)?